contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -39,11 +41,15 @@ install() {
 	dracut_install hostid
 	dracut_install awk
 	dracut_install head
+	dracut_install sort
+	dracut_install grep
 	inst_hook cmdline 95 "${moddir}/parse-zfs.sh"
 	inst_hook mount 98 "${moddir}/mount-zfs.sh"
 	inst_hook shutdown 30 "${moddir}/export-zfs.sh"
 
-	inst_simple "${moddir}/zfs-lib.sh" "/lib/dracut-zfs-lib.sh"
+	inst_simple "@sysconfdir@/zfs/zfs-functions" "/lib/dracut-zfs-lib.sh"
+	inst "@initconfdir@/zfs"
```

Comment:
Bug. This file is not parsed by `make` to parse out `initconfdir`. It parses out `sysconfdig` but not `initconfdir`.